sos/report/plugins/tpm2.py

```diff
@@ -0,0 +1,29 @@
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.report.plugins import Plugin, IndependentPlugin
+
+class TPM2(Plugin, IndependentPlugin):
+    """
+    Collects information about TPM2 module installed on host system.
+
+    This plugin will capture data using tpm2_tools
+    """
+
+    short_desc = 'Trusted Platform Module 2.0'
+    plugin_name = 'tpm2'
+    profiles = ('security', 'system', 'storage', 'hardware')
+    packages = ('tpm2-tools')
```

Comment:
This needs to be a tuple - as written this is a string. Single-item tuples still need a comma, e.g. `('tpm2-tools',)`.

Reply:
Got it done.